packages/samples/headless-ssr/cypress/e2e/ssr-search-parameter-manager.cy.ts

```diff
@@ -19,17 +19,20 @@ routes.forEach((route) => {
           .as('initial-results');
       });
 
-      it('should not update the search parameters', () => {
-        cy.url().should((href) =>
-          expect(new URL(href).searchParams.size).to.equal(0)
-        );
+      it('should add the tab search parameter with the default value', () => {
```

Comment:
Had to modify the tests since the tab query param gets added automatically when the tab manager initializes